ui/js/directives/treeherder/clonejobs.js

```diff
@@ -788,11 +788,23 @@ treeherder.directive('thCloneJobs', [
                 } else if (window.innerHeight >= 1000) {
                     scrollOffset = -200;
                 }
-                $('.th-global-content').scrollTo(el, duration, {offset: scrollOffset});
+                if(!el.isOnScreen()) {
+                    $('.th-global-content').scrollTo(el, duration, {offset: scrollOffset});
+                }
             }
 
         };
 
+        $.fn.isOnScreen = function(){
```

Comment:
We don't tend to use prototype functions in our code.  For consistency, could we perhaps just change this to:

    var isOnScreen = function(el) {

This would just be more consistent with the rest of the codebase.